app/client/lib/ethereum/observeCustomContracts.js

```diff
@@ -33,8 +33,6 @@ var addLogWatching = function(newDocument){
         if(!error) {
             var id = Helpers.makeId('log', web3.sha3(log.logIndex + 'x' + log.transactionHash + 'x' + log.blockHash));
 
-            Helpers.eventLogs(log);
```

Comment:
Having a log for every event detected would flood the console.log when observing a very active contract